### DISCUSSION TITLE
magpie与reshade工作区别

### DISCUSSION THREAD

**Discussion by JohnnyJosda** [Blinue/Magpie#819] [Q&A] [answered]
比如现在两个都打开CAS锐化，我能感觉Magpie增加了延迟，但是Reshade就没有这种感觉，两个工作原理是有区别的吗？

**Comment by Blinue** [comment, marked as answer]
Reshade 注入游戏进程修改画面，而 Magpie 是捕获然后放大，所以 Magpie 延迟更高。一般延迟为一帧，如果开启了允许额外的延迟则为两帧。
